lib/logstash/inputs/tcp.rb

```diff
@@ -136,6 +110,7 @@ def run_server(output_queue)
         # start a new thread for each connection.
         server_connection_thread(output_queue, socket)
       rescue OpenSSL::SSL::SSLError => e
+        # TODO: (colin) not sure I undestand the logic of logging & ignoring ssl errors here
```

Comment:
my understanding is that if a user connects with an incorrect ssl setup, an exception is raised on the server side (here), but the only thing we care about is dropping this client's connection and accept the next one